src/test/regress/sql/pg16.sql

```diff
@@ -623,22 +604,13 @@ SELECT roleid::regrole::text AS role, member::regrole::text,
 admin_option, inherit_option, set_option FROM pg_auth_members
 WHERE roleid::regrole::text = 'role1' ORDER BY 1, 2;
 
-SET citus.enable_ddl_propagation TO off;
-GRANT role1 TO role2 WITH ADMIN OPTION, INHERIT FALSE, SET FALSE;
-RESET citus.enable_ddl_propagation;
-
-SELECT roleid::regrole::text AS role, member::regrole::text,
-admin_option, inherit_option, set_option FROM pg_auth_members
-WHERE roleid::regrole::text = 'role1' ORDER BY 1, 2;
-
 \c - - - :master_port
-REVOKE role1 FROM role2;
 
 -- test REVOKES as well
 GRANT role1 TO role2;
 REVOKE SET OPTION FOR role1 FROM role2;
 REVOKE INHERIT OPTION FOR role1 FROM role2;
-
+\c - - - :master_port
```

Comment:
We are already connected to the coordinator at this point. I believe we can remove this line.

```suggestion
```